h/session.py

```diff
@@ -47,21 +48,43 @@ def _current_groups(request):
     This list is meant to be returned to the client in the "session" model.
 
     """
-    groups = [
-        {'name': 'Public', 'id': '__world__', 'public': True},
-    ]
+
     user = request.authenticated_user
+    authority = user.authority if user else request.auth_domain
```

Comment:
Correct on both counts. I haven’t yet tweaked this to accommodate unauthenticated users (and the `authority` parameter that goes along with it), which is why the `effective_principals` is still in place on the view.